client/src/components/Alert/Alert.tsx

```diff
@@ -68,7 +68,7 @@ const Alert: React.FC<AlertProps> = ({
         horizontal
       }}
       open={open}
-      autoHideDuration={autoHideDuration}
+      autoHideDuration={open ? autoHideDuration : undefined}
```

Comment:
Please, explain me this ternary operator in comment and tell me if using && operator is impossible for this case

`autoHideDuration={open && autoHideDuration}`

Reply:
@mras2303 
i've try many things but only this works :D
I did this becouse i noticed that CLER_ALERT actions i dispatching sometimes randomly

Reply:
`autoHideDuration={open && autoHideDuration}`
i tried but i got error from types that this prop can't be false